x-pack/plugins/fleet/server/services/epm/packages/get.test.ts

```diff
@@ -221,16 +221,16 @@ describe('When using EPM `get` services', () => {
         })
       ).resolves.toMatchObject([
         {
+          id: 'elasticsearch',
           name: 'elasticsearch',
           version: '0.0.1',
           title: 'Elasticsearch',
-          status: 'upload',
           savedObject: {
             id: 'elasticsearch',
             attributes: {
               name: 'elasticsearch',
               version: '0.0.1',
-              install_status: 'upload',
+              install_source: 'upload',
```

Comment:
Was this a wrong parameter or did it recently change?

Reply:
it was a typo in the test, I meant to use `install_source`.